openshift-with-appstudio-test/e2e/jvm_build_service_test.go

```diff
@@ -374,6 +375,44 @@ func TestExampleRun(t *testing.T) {
 		}
 	})
 
+	ta.t.Run("taskruns related to artifactbuilds and dependencybuilds generated", func(t *testing.T) {
```

Comment:
did you add this on purpose @stuartwdouglas or did this reappear as part of my attempt to provide rebase help?

this test may work in CI because we turn off pruning but can fail for folks testing locally if they do not turn off pruning

I would suggest just keeping this removed, unless you felt strongly about it

Reply:
I think that is a rebase issue. I will fix.